### PR TITLE
support django 3.0

### PR DESCRIPTION
**Pull request by jimothyGator**
Similar to #39, but sets django < 3.1, and adds Django 3.0 to tox file.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -14,10 +15,13 @@ setenv =
     PIP_INDEX_URL = https://pypi.python.org/simple/
 deps =
     py27: mock
+    py27: pyhamcrest==1.9.0
```
Comment (jimothyGator) [inline tox.ini]: Travis build failed when using pyhamcrest 1.10.0 with python 2.7. I don't fully understand why, but using pyhamcrest 1.9 fixed the build.

**Comment by jimothyGator** [comment]
Closes #38

**Comment by belugame** [comment]
@mattLLVW sorry, i could only merge one and the tox coverage is important for us

**Comment by mattLLVW** [comment]
@belugame Seriously dude?
There's a high security issue on Django 3.0, it's useless

**Comment by belugame** [comment]
@mattLLVW  i wasn't aware of a security issue. feel free to extend the version range to include 3.1. just make sure to also build it on tox.

**Comment by jimothyGator** [comment]
@belugame The security issue was specific to 3.0.0 was fixed in 3.0.1. 3.0.2 is the current version; there's not yet a 3.1.

@mattLLVW Sorry for the confusion; I should have specified that my PR allows 3.0.x, not 3.0.0 specifically. So when Django 3.0.3, 3.0.4, etc. are released, this wouldn't require a new PR for django-currentuser. It's only when Django 3.1 comes out that a new PR would be needed.

In retrospect, I should have commented on your PR suggesting a change to `tox.ini` rather than creating my own PR. I hope no offense was taken, as I surely didn't mean it.

https://www.djangoproject.com/weblog/2019/dec/18/security-releases/

**Comment by jimcushing-natgeo** [comment]
@belugame Thanks for merging this. Will you be able to release 0.4.3 to pypi soon?

**Pull request merged**